api/src/opentrons/protocols/context/protocol_api/protocol_context.py

```diff
@@ -273,10 +273,13 @@ def delay(self, seconds: float, msg: Optional[str]) -> None:
         """Delay execution for x seconds."""
         self._hw_manager.hardware.delay(seconds)
 
-    def home(self) -> None:
+    def home(self, home_plungers: bool = True) -> None:
         """Home the robot."""
         self.set_last_location(None)
-        self._hw_manager.hardware.home()
+        if not home_plungers:
+            self._hw_manager.hardware.home([Axis.X, Axis.Y, Axis.Z, Axis.A])
+        else:
+            self._hw_manager.hardware.home()
```

Comment:
This is a nitpick and I don't know if anyone else agrees, so feel free to ignore this suggestion, but I usually find double-negative `else` statements harder to read. Would this be clearer like this?

```suggestion
        if home_plungers:
            self._hw_manager.hardware.home()
        else:
            self._hw_manager.hardware.home([Axis.X, Axis.Y, Axis.Z, Axis.A])
```